x-pack/plugins/endpoint/public/applications/endpoint/store/policy_details/selectors.ts

```diff
@@ -119,6 +119,26 @@ export const selectedMacEvents = (state: PolicyDetailsState): number => {
   return 0;
 };
 
+/** Returns the total number of possible linux eventing configurations */
+export const totalLinuxEvents = (state: PolicyDetailsState): number => {
+  const config = policyConfig(state);
+  if (config) {
+    return Object.keys(config.linux.events).length;
+  }
+  return 0;
+};
+
+/** Returns the number of selected liinux eventing configurations */
+export const selectedLinuxEvents = (state: PolicyDetailsState): number => {
+  const config = policyConfig(state);
+  if (config) {
+    return Object.values(config.linux.events).reduce((count, event) => {
```

Comment:
:grey_question:  `Object.values(config.linux.events).filter(event=>event===true).length` might be a little easier to read